plugin/evm/static_service.go

```diff
@@ -8,15 +8,14 @@ import (
 	"encoding/json"
 
 	"github.com/ava-labs/coreth/core"
-	"github.com/ava-labs/avalanchego/utils/formatting"
 )
 
 // StaticService defines the static API services exposed by the evm
 type StaticService struct{}
 
 // BuildGenesis returns the UTXOs such that at least one address in [args.Addresses] is
 // referenced in the UTXO.
-func (*StaticService) BuildGenesis(_ context.Context, args *core.Genesis) (formatting.CB58, error) {
+func (*StaticService) BuildGenesis(_ context.Context, args *core.Genesis) ([]byte, error) {
```

Comment:
Should this return `{bytes, encoding}`?

Reply:
Updated to return a BuildGenesisReply struct.